docs-2.0/3.ngql-guide/14.index-statements/1.create-index.md

```diff
@@ -1,27 +1,56 @@
 # CREATE INDEX
 
+Use `CREATE INDEX` to add indexes to existing tags, edge types or properties.
```

Comment:
where is the `create index` syntax?

"to add an index for existing properties on a tag or an edge type"?

Reply:
Now it's not only for properties. We can add an index for tags or edge types. But for is better than to. Thank you.